static/js/map.js

```diff
@@ -6847,24 +6847,12 @@ $(function () {
             Store.set('remember_select_exclude_min_iv', excludedMinIV)
         })
         $textMinIV.on('change', function (e) {
-            minIV = parseInt($textMinIV.val(), 10)
-            if (isNaN(minIV) || minIV < 0) {
-                minIV = ''
-            }
-            if (minIV > 100) {
-                minIV = 100
-            }
+            minIV = Math.max(0, Math.min(parseInt($textMinIV.val(), 10) || 0, 100))
             $textMinIV.val(minIV)
             Store.set('remember_text_min_iv', minIV)
         })
         $textMinLevel.on('change', function (e) {
-            minLevel = parseInt($textMinLevel.val(), 10)
-            if (isNaN(minLevel) || minLevel < 0) {
-                minLevel = ''
-            }
-            if (minLevel > 35) {
-                minLevel = 35
-            }
+            minLevel = Math.max(1, Math.min(parseInt($textMinLevel.val(), 10) || 1, 35))
```

Comment:
This would force the min level filter to be at least 1, while 0 is to disable.
`minLevel = Math.max(0, Math.min(parseInt($textMinLevel.val(), 10) || 0, 35))`

Reply:
Oh I hadn't considered that "0" is probably the default level for unscanned pokemon. That can easily be swapped